static/src/javascripts/projects/common/modules/ui/autoupdate.js

```diff
@@ -176,7 +176,8 @@ define([
 
             this.notificationBar = new NotificationBar({attachTo: $('.js-update-notification')[0] });
 
-            $(options.attachTo).addClass('autoupdate--has-animation');
+            var identity = function (val) { return val; };
+            $(options.attachTo.filter(identity)).addClass('autoupdate--has-animation');
```

Comment:
is `identity` the best way to check the validity of `attachTo`? Seems a bit...javascript-y :)